options/locale/locale_en-US.ini

```diff
@@ -185,13 +185,13 @@ network_error = Network error
 [startpage]
 app_desc = A painless, self-hosted Git service
 install = Easy to install
-install_desc = Simply <a target="_blank" rel="noopener noreferrer" href="https://docs.gitea.com/installation/install-from-binary">run the binary</a> for your platform, ship it with <a target="_blank" rel="noopener noreferrer" href="https://github.com/go-gitea/gitea/tree/master/docker">Docker</a>, or get it <a target="_blank" rel="noopener noreferrer" href="https://docs.gitea.com/installation/install-from-package">packaged</a>.
+install_desc = Simply %[1]srun the binary%[2]s for your platform, %[3]sship it with Docker%[4]s, or get it %[5]spackaged%[6]s.
```

Comment:
TBH I do not think these changes are good. `%[1]srun` makes it more difficult to translators, and it's impossible to correctly spell-check it.

And `SafeHTML` should be avoided.